statesman-server/src/main/java/io/appform/statesman/server/dao/message/MessageConfigStoreCommand.java

```diff
@@ -0,0 +1,73 @@
+package io.appform.statesman.server.dao.message;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.LoadingCache;
+import io.appform.dropwizard.sharding.dao.LookupDao;
+import io.appform.statesman.engine.MessageConfigStore;
+import io.appform.statesman.model.MessageConfig;
+import io.appform.statesman.model.exception.ResponseCode;
+import io.appform.statesman.model.exception.StatesmanError;
+import io.appform.statesman.server.utils.MapperUtils;
+import io.appform.statesman.server.utils.WorkflowUtils;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@Singleton
+public class MessageConfigStoreCommand implements MessageConfigStore {
+
+    private final LookupDao<StoredMessageConfig> messageLookupDao;
+    private final LoadingCache<String, Optional<MessageConfig>> messageConfigCache;
+
+    @Inject
+    public MessageConfigStoreCommand(LookupDao<StoredMessageConfig> messageLookupDao) {
+        this.messageLookupDao = messageLookupDao;
+        messageConfigCache = Caffeine.newBuilder()
+                .maximumSize(1_000)
+                .expireAfterWrite(300, TimeUnit.SECONDS)
+                .refreshAfterWrite(60, TimeUnit.SECONDS)
```

Comment:
whats expire after write ? also refresh can be slower and 15 mins should be fine IMO

Reply:
expire after write is basically looks at last write time and expires the entry after specified time, we can increase it and sure I will increase the refresh time